docs/README.md

```diff
@@ -1,12 +1,10 @@
 # Environment setup
 
-You have 3 options for environment setup, installation of APIs and using Notebooks. Please choose the one which suits you best.
+You have 2 options for environment setup and using Notebooks. Please choose the one which suits you best.
 
-1. [**Github CodeSpaces**](environment-setup-codespaces.md) - a purely cloud-based environment.  This should be your go-to option if you do NOT have a linux environment (OSX or WSL/Windows) or docker on your laptop/workstation.
```

Comment:
This file https://github.com/MS-SJF-Projects/agent-lab/blob/main/docs/environment-setup-codespaces.md could also be removed now.

Reply:
removed